README.md

```diff
@@ -123,6 +123,8 @@ window.setTimeout(function () {
 }, splashDuration - fadeDuration);
 ```
 
+### iOS Quirks
```

Comment:
@daserge  I have just changed document for Android only. You can see by expanding those full changes.